express-error-handling/server.ts

```diff
@@ -0,0 +1,114 @@
+import express from 'express';
+import { errorMiddleware, ClientError } from './lib/index.js';
+
+type Note = {
+  id: number;
+  content: string;
+};
+
+const app = express();
+
+app.get('/api/notes', async (req, res, next) => {
+  try {
+    const notes = await readNotes();
+    res.send(notes);
+  } catch (err) {
+    res.status(500).send({ error: 'an unexpected error occurred' });
```

Comment:
Remove this line. It's handled by the errorHandlingMiddleware. Do this on all your routes.